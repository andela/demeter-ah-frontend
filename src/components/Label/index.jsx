import React from 'react';

const Label = (props) => {
  const {
    name, labelname, classes, ...rest
  } = props;

  return (
    <label htmlFor={name} {...rest} className={classes}>
      {labelname || name}
    </label>
  );
};

export default Label;
