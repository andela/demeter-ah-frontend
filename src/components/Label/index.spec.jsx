import React from 'react';
import { shallow } from 'enzyme';
import Label from './index';

describe('Label Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<Label debug />);
    expect(component).toMatchSnapshot();
  });
});
