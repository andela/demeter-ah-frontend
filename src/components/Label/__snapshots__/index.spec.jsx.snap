// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component Should render without errors 1`] = `
<label
  debug={true}
/>
`;
